src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -230,7 +229,22 @@
       <OldBuildVersionFiles Include="$(BuildVersionFilePath)BuildVersions-*.props" Exclude="$(BuildVersionFilePath)%(BuildVersionFileItem.Filename).props" />
     </ItemGroup>
     <Delete Files="@(OldBuildVersionFiles)" TreatErrorsAsWarnings="true"/>
+  </Target>
+
+  <Target Name="GenerateBuiltByString" Condition="'$(BuiltByString)'==''">
+
+    <!-- If building on Windows, default BuiltByString to use Computername and username -->
+    <BuiltByString Condition="'$(OSEnvironment)'=='Windows_NT'">%20built by: $(COMPUTERNAME)-$(USERNAME)</BuiltByString>
```

Comment:
Oh I see, sorry I didn't understand the first time. I just pushed a new change that hopefully does what you expected. Let me know otherwise.